### ISSUE TITLE
Interface linkstate and speed behaviour changes in JSON

### ISSUE THREAD

**Issue by andydavidson** [napalm-automation-community/napalm-cumulus#23]
Issue number 3 detailed a mapping between interface linkstate and speed which no longer seems to be in place:

cumulus@cumulus:~$ net show version 
NCLU_VERSION=1.0
DISTRIB_ID="Cumulus Linux"
DISTRIB_RELEASE=3.5.2
DISTRIB_DESCRIPTION="Cumulus Linux 3.5.2"

```
"swp1":{

    "iface_obj":{
        "native_vlan":null,
        "description":"",
        "vlan":[
            {
                "vlan":1,
                "flags":[
                    "PVID",
                    "Egress Untagged"
                ]
            }
        ],
        "lacp":{
            "rate":"",
            "sys_priority":"",
            "partner_mac":"",
            "bypass":""
        },
        "mac":"00:e0:ec:7b:72:ee",
        "members":{
        },
        "min_links":"",
        "dhcp_enabled":false,
        "ip_address":{
            "allentries":[
            ]
        },
        "ip_neighbors":null,
        "lldp":null,
        "asic":{
            "asicname":"xe0.0"
        },
        "mtu":1500,
        "vlan_filtering":true,
        "counters":{
            "rx":{
                "unicast":0,
                "broadcast":3,
                "multicast":2,
                "errors":0
            },
            "tx":{
                "unicast":0,
                "broadcast":0,
                "multicast":176,
                "errors":0
            }
        }
    },
    "linkstate":"UP",
    "summary":"",
    "connector_type":"Unknown",
    "mode":"Access/L2",
    "speed":"1G"
}
```

I will create a pull request that loads the napalm interface information from teh right bit of the cumulus JSON now.

**Comment by dbarrosop** [comment]
> I notice you changed the version in the setup.py file. I'm not sure what the versioning scheme

In 0.x.y we usually change `y` for bugfixes and `x` when adding new stuff but feel free to follow any other format if you prefer.

**Comment by jmrenshaw** [comment]
Closed in PR #23
